src/plugins/console/server/lib/spec_definitions/json/generated/cat.cluster_manager.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
For my own learning, is there documentation for the steps for updating these generated definitions?

Reply:
Yes! Thanks to @kavilla  and @CPTNB, we now have it published here - https://cptnb.github.io/opensearch-dashboards-dev-docs/core-plugins/console/console/